common/app/views/support/CamelCase.scala

```diff
@@ -7,4 +7,8 @@ object CamelCase {
         first + rest.map(_.capitalize).mkString("")
       case Nil => ""
     }
+
+  private val lowerCaseFollowedByUpperCase = """([a-z])([A-Z])""".r
+  def toHyphenated(s: String): String =
+    lowerCaseFollowedByUpperCase.replaceAllIn(s, m => m.group(1) + "-" + m.group(2)).toLowerCase
```

Comment:
is this a reasonable place to put this? It's essentially the inverse of the `fromHyphenated`. Implementation is grabbed from capi-models https://github.com/guardian/content-api-models/blob/main/json/src/main/scala/com/gu/contentapi/json/CirceEncoders.scala#L15-L21

Reply:
Looks very reasonable to me!